docs/architecture/psa-migration/psa-limitations.md

```diff
@@ -14,8 +14,8 @@ Limitations relevant for G1 (performing crypto operations)
 Executive summary
 -----------------
 
-- Restartable/interruptible ECC operations: in progress (mid-2024).
-- Arbitrary parameters for FFDH: will be dropped in 4.0.
+- Restartable/interruptible ECC operations: support in PSA will be added in 4.0.
```

Comment:
“will be added in 4.0” isn't quite accurate. Interruptible signatures were already in 2.28. Restartable ECDH should be in 4.0. We haven't decided yet about restartable export-public. Their usage in TLS will not be in 4.0.